### ISSUE TITLE
4. An idea for an improvement/clarification could be: (explain what and why)

### ISSUE THREAD

**Issue by Nicolesc1** [NumEconCopenhagen/projects-2024-hwee-oscar-thomas#5]
I think overall, everything is done really well. I think just better descriptions of the steps of why you are doing everything would have been good. This is because it is a bit hard to follow the code, as I am not sure why you did certain things in those ways.

**Comment by fpv111** [comment]
I don't think it is necessary for you, to improve your code. It is already really good. But you could improve it some places, by writing more explaining text. 

In Q4 you have written:" For question 4b: Seems like a wrong result.". I think it might be a mistake, and just a comment for yourself?

**Comment by drp193** [comment]
When handing in the exam paper, I think it would be beneficial to add parts of the code you have in the notebook to the model .py file (as mentioned before). I think that would make a "nicer" and simpler results page. Other than that, not much to add which hasn't been mentioned.

**Comment by Jiyunkimmm** [comment]
Also in question 6b # For demonstration, replace these with your actual results seems not that necessary. Except these #lines, the other codes are perfect.

**Comment by cmv534** [comment]
I believe that the code itself is very neat and well structured. I would probably add on the notebook small descriptions so that the economic intuition behind the computations made is more clear to the reader.
